validator/accounts/accounts_activation.go

```diff
@@ -0,0 +1,270 @@
+	case keymanager.Imported:
+		km, ok := ikeymanager.(*imported.Keymanager)
+		if !ok {
+			return errors.New("not a imported keymanager")
+		}
+		disabledPublicKeys := make([][48]byte, len(km.KeymanagerOpts().DisabledPublicKeys))
+		for i, dpk := range km.KeymanagerOpts().DisabledPublicKeys {
+			disabledPublicKeys[i] = bytesutil.ToBytes48(dpk)
+		}
+		if len(disabledPublicKeys) == 0 {
+			return errors.New("No accounts are disabled.")
+		}
+		// Allow the user to interactively select the accounts to enable or optionally
+		// provide them via cli flags as a string of comma-separated, hex strings.
+		filteredPubKeys, err := filterPublicKeysFromUserInput(
+			cliCtx,
+			flags.EnablePublicKeysFlag,
+			disabledPublicKeys,
+			prompt.SelectAccountsEnablePromptText,
+		)
+		if err != nil {
+			return errors.Wrap(err, "could not filter public keys for activation")
+		}
+		rawPublicKeys := make([][]byte, len(filteredPubKeys))
+		formattedPubKeys := make([]string, len(filteredPubKeys))
+		for i, pk := range filteredPubKeys {
+			pubKeyBytes := pk.Marshal()
+			rawPublicKeys[i] = pubKeyBytes
+			formattedPubKeys[i] = fmt.Sprintf("%#x", bytesutil.Trunc(pubKeyBytes))
+		}
+		allAccountStr := strings.Join(formattedPubKeys, ", ")
+		if !cliCtx.IsSet(flags.DisablePublicKeysFlag.Name) {
+			if len(filteredPubKeys) == 1 {
+				promptText := "Are you sure you want to enable 1 account? (%s) Y/N"
+				resp, err := promptutil.ValidatePrompt(
+					os.Stdin, fmt.Sprintf(promptText, au.BrightGreen(formattedPubKeys[0])), promptutil.ValidateYesOrNo,
+				)
+				if err != nil {
+					return err
+				}
+				if strings.ToLower(resp) == "n" {
+					return nil
+				}
+			} else {
+				promptText := "Are you sure you want to enable %d accounts? (%s) Y/N"
+				if len(filteredPubKeys) == len(disabledPublicKeys) {
+					promptText = fmt.Sprintf("Are you sure you want to enable all accounts? Y/N (%s)", au.BrightGreen(allAccountStr))
+				} else {
+					promptText = fmt.Sprintf(promptText, len(filteredPubKeys), au.BrightGreen(allAccountStr))
+				}
+				resp, err := promptutil.ValidatePrompt(os.Stdin, promptText, promptutil.ValidateYesOrNo)
+				if err != nil {
+					return err
+				}
+				if strings.ToLower(resp) == "n" {
+					return nil
+				}
+			}
+		}
+		if err := EnableAccount(cliCtx.Context, &AccountConfig{
+			Wallet:     w,
+			Keymanager: ikeymanager,
+			PublicKeys: rawPublicKeys,
+		}); err != nil {
+			return err
+		}
+		log.WithField("publicKeys", allAccountStr).Info("Accounts enabled")
+		return nil
+	case keymanager.Derived:
+		return errors.New("cannot enable accounts for a derived keymanager")
+	default:
+		return fmt.Errorf("keymanager kind %s not supported", w.KeymanagerKind())
+	}
+}
+
+// DisableAccount disable the accounts that the user requests to be disabled from the wallet
+func DisableAccount(ctx context.Context, cfg *AccountConfig) error {
+	switch cfg.Wallet.KeymanagerKind() {
+	case keymanager.Remote:
```

Comment:
For now, I only implemented the feature for Imported accounts.
I assume we also want to support disabling/enabling accounts for Remote/Derived but waiting for confirmation

Reply:
@rauljordan do you confirm that we also want the enable/disable feature for Remote and Derived keymanager?

Reply:
We do not want one for remote. Derived should also be nice, but can be done in as separate PR